### DISCUSSION TITLE
May I request a patch version release?

### DISCUSSION THREAD

**Discussion by LukasParke** [alex8088/electron-vite#432] [General]
Hi @alex8088! 

I recently submitted a one line PR to address the use case of using SvelteKit as the renderer in the `electron-vite` config.
I am looking to use the `electron-vite` package in several production applications, but until a version with that PR is released I am not able migrate those projects over.

If you can release a patch version of the package including that PR I would very much appreciate it!

Thank you.

**Comment by alex8088** [comment]
It will be released in the next few days, and there are still some issues to be fixed.

**Reply by LukasParke** [reply]
Thank you Alex!

**Comment by alex8088** [comment]
new ver 2.1.0 is out 🎉.
